### ISSUE TITLE
Docs build fails

### ISSUE THREAD

**Issue by DanielKotik** [mala-project/mala#299]
![grafik](https://user-images.githubusercontent.com/28047702/160199452-aa687ede-7c6d-43ce-96e8-3556ff16934b.png)

**Comment by RandomDefaultUser** [comment]
Yep, I have seen this one. I will investigate. I have no clue what's going on, since the error started appearing on a commit which didn't even change the documentation!

**Comment by DanielKotik** [comment]
I've also looked at this, but haven't found a quick solution.

**Comment by RandomDefaultUser** [comment]
As the error is related to jinja, and jinja released a new version on Thursday (the first day, the doc started failing) I am going to go out on a limb here and think it's probably that.

**Comment by RandomDefaultUser** [comment]
This may actually a good point to also tackle #279. I bet the new jinja version introduces an incompatibility that the more recent sphinx versions have already taken care of.

**Comment by DanielKotik** [comment]
Well you're probably right. If I use a `venv` and install packages specified in `dpcs/requirements.txt` I get the same issue. If I create a new environment based on a `requirements.txt` with following content:

```
docutils
sphinx
sphinx_rtd_theme
myst-parser
sphinx-markdown-tables
```
the docs can be build again, but with a warning (which results in an error and thus a failing pipeline):

```
writing output... [100%] usage/trainingmodel                                                                                                                                                                                                                 
/home/kotik/GITHUB/work/mala_fork/docs/source/CONTRIBUTE.md:10: WARNING: 'myst' reference target not found: #branching-strategy
/home/kotik/GITHUB/work/mala_fork/docs/source/CONTRIBUTE.md:46: WARNING: 'myst' reference target not found: #developing-code
generating indices... genindex py-modindex erledigt
highlighting module code... [100%] mala.targets.target                                                                                                                                                                                                       
writing additional pages... search erledigt
copying images... [100%] ../../docs/source/img/logos/mala_horizontal.png                                                                                                                                                                                     
copying static files... erledigt
copying extra files... erledigt
dumping search index in English (code: en)... erledigt
dumping object inventory... erledigt
build mit Problemen beendet, 2 warnings.
```

A `pip freeze > requirements.txt` gives

```
alabaster==0.7.12
attrs==21.4.0
Babel==2.9.1
certifi==2021.10.8
charset-normalizer==2.0.12
docutils==0.17.1
idna==3.3
imagesize==1.3.0
importlib-metadata==4.11.3
Jinja2==3.1.1
Markdown==3.3.6
markdown-it-py==2.0.1
MarkupSafe==2.1.1
mdit-py-plugins==0.3.0
mdurl==0.1.0
myst-parser==0.17.0
packaging==21.3
Pygments==2.11.2
pyparsing==3.0.7
pytz==2022.1
PyYAML==6.0
requests==2.27.1
snowballstemmer==2.2.0
Sphinx==4.5.0
sphinx-markdown-tables==0.0.15
sphinx-rtd-theme==1.0.0
sphinxcontrib-applehelp==1.0.2
```

**Comment by DanielKotik** [comment]
I've tried using `docutils==0.16` instead of `docutils==0.17.1` (we had some issues with `docutils` before) but this does not fix the `WARNING: 'myst' reference target not found` issue.

**Comment by RandomDefaultUser** [comment]
I was able to fix both this error and the sphinx error in #300 :)

**Comment by RandomDefaultUser** [comment]
Closed with #301.
